src/node_crypto.cc

```diff
@@ -1974,7 +1974,78 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
 }
 
 
-// TODO(indutny): Split it into multiple smaller functions
+// Put issuer inside the object
```

Comment:
Not a hugely useful comment.  If you call the function `AddIssuerToObject()`, it's self-documenting.

Reply:
I like that, I'll rename the function.